applications/nrf_desktop/doc/hid_reportq.rst

```diff
@@ -0,0 +1,81 @@
+.. _nrf_desktop_hid_reportq:
+
+HID report queue utility
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+A HID subscriber (for example, a USB HID class instance handled by :ref:`nrf_desktop_usb_state`) limits the maximum number of HID input reports that can be handled simultaneously.
+Enqueuing reports locally at the source may be necessary to prevent HID report drops.
+The HID report queue utility can be used by an application module to simplify enqueuing HID input reports received from connected HID peripherals before providing them to HID subscriber.
+
+Configuration
+*************
+
+Make sure that heap size (:kconfig:option:`CONFIG_HEAP_MEM_POOL_SIZE`) is large enough.
```

Comment:
Changed to:
```
Make sure that heap size (:kconfig:option:`CONFIG_HEAP_MEM_POOL_SIZE`) is large enough to handle the worst possible use case.

```

Reply:
Still - please provide some number or estimate. In other case IMO we should remove this sentence.

Reply:
I am worried that it might not be possible to provide numbers/estimates here (it depends on use-case). We already use similar approach here: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/applications/nrf_desktop/description.html#memory_allocation

Reply:
Maybe `Calculate the heap size in worst possible use case and set (:kconfig:option:`CONFIG_HEAP_MEM_POOL_SIZE`) according to that.``

IMO "Make sure" - is a phrase I would not bother with when reading instruction while some verb is a better call to action.

Reply:
Actually in most cases you `estimate` the heap size by making some practical tests (generally we do not `calculate` it). Also, it's possible that default value is ok and update is not needed - that's why I used `make sure` here (as we do in other places in documentation)